src/Apps/Artist/client.ts

```diff
@@ -0,0 +1,64 @@
+// import { buildClientApp } from "reaction/Artsy/Router/client"
```

Comment:
This is a straight import of Force’s client file, so this needs to be made to work, which requires a working webpack setup. Where possible, it would be nice to share config with the storybook setup.

Reply:
To be able to test and track the performance of these apps on CI